examples/landing-zone-v2/tier3/external-load-balancer/address.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# reserve an external IP address for the load balancer
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeAddress
+metadata:
+  name: workload-name-external-ip # kpt-set: ${workload-name}-external-ip
+  annotations:
+    cnrm.cloud.google.com/project-id: project-id # kpt-set: ${project-id}
+spec:
+  resourceID: workload-name-external-ip # kpt-set: ${workload-name}-external-ip
+  description: external IP for workload-name # kpt-set: external IP for ${workload-name}
+  addressType: EXTERNAL
```

Comment:
I would like to see the key : `networkTier` included in this example.

Reply:
for some reason, adding `networkTier: PREMIUM` caused config sync to fight with itself

```Error:   KNV2005: detected excessive object updates, approximately 27 times per minute. This may indicate Config Sync is fighting with another controller over the object.```